website/source/docs/provisioners/file.html.md

```diff
@@ -30,17 +30,16 @@ The file provisioner can upload both single files and complete directories.
 
 ## Configuration Reference
 
-The available configuration options are listed below. All elements are required.
+The available configuration options are listed below.
+
+### Required
 
 -   `source` (string) - The path to a local file or directory to upload to
     the machine. The path can be absolute or relative. If it is relative, it is
     relative to the working directory when Packer is executed. If this is a
     directory, the existence of a trailing slash is important. Read below on
     uploading directories.
 
--   `sources` (array of strings) - A list of paths or directories to upload to
-    the machine. This is similar to the `source` option.
-
```

Comment:
Hi @vtolstov, 

Good question. If I'm not mistaken, it looks like the `sources` config option is [something you implemented](https://github.com/mitchellh/packer/blob/feee19e4ed51e9045e4de6639b4b954f5ac37e58/provisioner/file/provisioner.go#L96) in Feb? It appears the docs for the feature may have been missed, which is why I added them here. I then removed them because I don't know what the intended behavior of `sources` is.

I'll explain why I feel this way. If I specify the following:

```json
{
    "sources": ["file1","path/file2"],
    "destination": "/tmp/"
}
```

I'd expect the files to get copied to the remote server like this:

1. /tmp/file1
2. /tmp/file2

What I get instead is this:

1. /tmp/tmp

Where /tmp/tmp contains the content of whichever file was copied last (/path/file2 in this case).

To be clear, I didn't remove the functionality of `sources`, I just continued to hide it as it always had been.

Good thing you raised it though because I was going to ask if it made sense to remove it or to improve it. If nothing else, I shouldn't be the one who documents it, since I don't know what its intended functionality was.